jsk_baxter_robot/baxtereus/baxter-interface.l

```diff
@@ -249,15 +249,17 @@
    (when (send self :simulation-modep)
      (return-from :angle-vector (send* self :angle-vector-raw av tm args)))
    (when (not (numberp tm))
-     (ros::warn ":angle-vector tm is not a number, use :angle-vector av tm args"))
+     (ros::warn ":angle-vector tm is not a number, use :angle-vector av tm args~%"))
    (if (and (get self :moveit-environment)
             (send (get self :moveit-environment) :robot))
      (if (or (eq move-arm :rarm) (eq move-arm :larm))
        (send self :angle-vector-motion-plan av :move-arm move-arm :total-time tm :start-offset-time 0.01 :use-torso nil :clear-velocities t)
-       (ros::warn ":angle-vector currently not support :arms, define :move-arm"))
+       (ros::warn ":angle-vector currently not support :arms, define :move-arm~%"))
      (progn
-       (ros::warn "moveit environment is not correctly set, execute :angle-vector-raw instead")
+       (ros::warn "moveit environment is not correctly set, execute :angle-vector-raw instead~%")
        (return-from :angle-vector (send* self :angle-vector-raw av tm args)))))
+  (:angle-vector-sequence (avs &rest args)
+		 (send* self :angle-vector-sequence-raw avs args))
```

Comment:
I think we can set multiple waypoint in planning. how about add multiple jointconstraint at https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/master/pr2eus_moveit/euslisp/robot-moveit.l#L224 ?